tools/plugins/com.liferay.ide.project.core/src/com/liferay/ide/project/core/modules/ServiceCommand.java

```diff
@@ -81,7 +82,6 @@
         "com.liferay.portal.service.http",
         "com.liferay.portal.service.permission",
         "com.liferay.portal.servlet",
-        "com.liferay.portal.servlet.filters.authverifier",
```

Comment:
For example, if we have:
"com.liferay.portal.servlet",
"com.liferay.portal.servlet.filters.authverifier",
that means all sub-packages of "com.liferay.portal.servlet" will be ignore, included "com.liferay.portal.servlet.filters.authverifier". so we no need to add the sub-packages if we already have the parent/root package in our list